### PR TITLE
feat(ui): add profile analytics view

### PR DESCRIPTION
**Pull request by wwayne**
image

image

Fix TAB-565

### CODE REVIEW

File: ee/tabby-ui/lib/types/stats.ts
Comment (wsxiaoys) [inline ee/tabby-ui/lib/types/stats.ts:1]: can we just reuse the type defined in gql? `DailyStatsQuery.dailyStats`

Comment (wwayne) [reply ee/tabby-ui/lib/types/stats.ts:1]: fixed, removed the type and start using the type from `@/lib/gql/generates`

File: ee/tabby-ui/app/(home)/components/stats.tsx
Comment (liangfung) [inline ee/tabby-ui/app/(home)/components/stats.tsx:1]: Perhaps move it directly into stats.tsx, until there is a need for reuse

File: ee/tabby-ui/app/(home)/use-language-stats.ts
Comment (liangfung) [inline ee/tabby-ui/app/(home)/use-language-stats.ts:1]: Perhaps move it directly into stats.tsx, until there is a need for reuse

Comment (wwayne) [reply ee/tabby-ui/app/(home)/use-language-stats.ts:1]: updated and merged the hook into the component

**Pull request merged**